### DISCUSSION TITLE
Changes to the About Section in GitHub after launching my page for the html portfolio

### DISCUSSION THREAD

**Discussion by MeghanEdnaW** [community/community#87711] [New to GitHub] [answered]
I have my page up, but made changes to the about section afterward, to replace the placeholder text. I changed it in VS Code, and in GitHub because I couldn't figure out how to replace the about section in my repository. I edited the about section, but only the placeholder text shows on my page. Please help,  and thank you. :)
![about section update](https://github.com/community/community/assets/156444466/be651971-09f2-4c7f-b9ee-2a4981fee5ee)
![Uploading about section update.jpg…]()

**Comment by mishmanners** [comment, marked as answer]
Did you commit and merge your changes to the main branch? Make sure you're editing the correct file - for what you're looking at, it's probably a html file under the "scr" folder or an index.html file under the repo. Also make sure the page you're viewing isn't cached, as that can sometimes show old versions of the website. Finally, ensure you've given a bit of time for the changes to flush through; up to 10 minutes should be plenty.

**Comment by MeghanEdnaW** [comment]
I don’t know how to do that. I took a screenshot of the changes to the file, which appear in green. In red, is the old placeholder text that shows on the page 

https://meghanednaw.github.io/html-portfolio/public/about.html

**Reply by mishmanners** [reply]
When you commit the code, are you committing to "main" or to a branch?

**Reply by MeghanEdnaW** [reply]
the main branch is all I'm using, I think. This is for school - its my first repository

**Comment by MeghanEdnaW** [comment]
Someone from Facebook helped navigate to the correct file to edit. There seems to be an order - one has to access the public folder, then the about file - rather than the about file directly- to display a change on the page. 
When I edited that file, my page updated right away.

**Reply by mishmanners** [reply]
Yes that's correct. You need to access the source file and commit to main. Glad you got it working 😊
